ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityMediationAdapter.java

```diff
@@ -14,17 +14,22 @@
 import com.google.android.gms.ads.mediation.MediationRewardedAdCallback;
 import com.google.android.gms.ads.mediation.MediationRewardedAdConfiguration;
 import com.google.android.gms.ads.mediation.VersionInfo;
+import com.unity3d.ads.BuildConfig;
+import com.unity3d.ads.IUnityAdsInitializationListener;
 import com.unity3d.ads.UnityAds;
 
 import java.util.HashSet;
 import java.util.List;
 
 /**
  * The {@link UnityMediationAdapter} is used to initialize the Unity Ads SDK, load rewarded
- * video ads from Unity ads and mediate the callbacks between Google Mobile Ads SDK and Unity Ads SDK.
+ * video ads from Unity Ads and mediate the callbacks between Google Mobile Ads SDK and Unity Ads SDK.
  */
-public class UnityMediationAdapter extends Adapter implements MediationRewardedAd {
```

Comment:
why do we remove `MediationRewardedAd `?

Reply:
Because showAd is in UnityRewardedAd now